bin/run.sh

```diff
@@ -32,16 +65,18 @@ SCITRAN_PERSISTENT_DATA_PATH=${SCITRAN_PERSISTENT_DATA_PATH:-"$SCITRAN_PERSISTEN
 SCITRAN_PERSISTENT_DB_PATH=${SCITRAN_PERSISTENT_DB_PATH:-"$SCITRAN_PERSISTENT_PATH/db"}
 SCITRAN_PERSISTENT_DB_PORT=${SCITRAN_PERSISTENT_DB_PORT:-"9001"}
 SCITRAN_PERSISTENT_DB_URI=${SCITRAN_PERSISTENT_DB_URI:-"mongodb://localhost:$SCITRAN_PERSISTENT_DB_PORT/scitran"}
-
 set +o allexport
 
+#Non-exported local variables
+PASTER_INI_FILE=$SCITRAN_PERSISTENT_PATH/paster.ini
+
 
-if [ -f "$SCITRAN_PERSISTENT_DB_PATH/mongod.lock" ]; then
+if [[ -f "$SCITRAN_PERSISTENT_DB_PATH/mongod.lock" ]]; then
```

Comment:
Using == is easier to read in my opinion. I changed all of them to be consistent in the file.